protocol/contact.go

```diff
@@ -26,6 +26,22 @@ type ContactDeviceInfo struct {
 	FCMToken string `json:"fcmToken"`
 }
 
+func (c *Contact) CanonicalName() string {
+	if c.LocalNickname != "" {
+		return c.LocalNickname
+	}
+
+	if c.ENSVerified {
+		return c.Name
+	}
+
+	return c.Alias
+}
+
+func (c *Contact) CanonicalImage() string {
+	return c.Identicon
+}
```

Comment:
Would a profile picture be considered a CanonicalImage? Or are we keeping them private?

Reply:
yes, I haven't fully implemented for now as I can't get them to display in push notification for some reason, but eventually the profile picture would be the canonical image (if present)